### PR TITLE
Port Canvas documentation to developers.optimizely.com

### PR DESCRIPTION
**Pull request by jgaul**
@jprovine @lucasoptimizely @jessicalc 

This is mostly a port of http://link.optimizely.com/canvas-docs, with some modifications. We may change the illustrative app to something other than Templates if Templates isn't part of the public beta launch, but leaving it for now.

### CODE REVIEW

File: src/partials/_nav.html [hunk 1]
```diff
@@ -3,47 +3,50 @@
...
     <ul class="unstyled nav nav--stacked soft-double delta">
       {% for key, item in paths.navigation %}
-        <li class="push--bottom">
-          <a class="main-section{% if item.path === pathArray[0] %} active{% endif %}" href="/{{ item.path }}"{% if item.path !== 'samples' && item.path !== 'overview' && item.path !== 'integrations' %} data-collapser{% endif %}>{{ item.title }}</a>
-          <!-- TOC for overview, integrations, and code samples -->
-          {% if item.path === 'overview' || item.path === 'integrations' %}
-            <ul class="unstyled nav nav--stacked epsilon toc"></ul>
-          {% endif %}
+        <!-- TODO: Show Canvas section when we launch the public beta -->
+        {% if item.path != 'canvas' %}
```
Comment (jgaul) [inline src/partials/_nav.html]: GitHub makes it look like I changes this file a lot more than I did. I just added this if block to exclude the Canvas section from the sidebar until the launch.

**Comment by jgaul** [comment]
@jprovine @lucasoptimizely @jessicalc boop

File: src/partials/_nav.html (hunk 1 above)
Comment (tscanlin) [inline src/partials/_nav.html]: You can also use the whitespace ignore param "?w=1"
https://github.com/optimizely/developers.optimizely.com/pull/263/files?w=1

Looking good!

Comment (jgaul) [inline src/partials/_nav.html]: cool tip! thank you, @tscanlin!

**Comment by jgaul** [comment]
@jprovine @jessicalc -- thank you for all the feedback! let me know whether this LGTY now!

**Comment by jessicalc** [comment]
LGTM, but I totally forgot to mention last time - could you put some kind
of Contact Info in these docs as well? Maybe as a section in the bottom?
Jess

**Comment by jessicalc** [comment]
One more thing - could you update the link to OUI to point to https://github.com/optimizely/oui/blob/devel/README.md ? I'll leave an inline comment. Right now it's not immediately clear how I can start using OUI based on those links

File: src/content/canvas/index.md [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+
+Per the `"expires_in": 7200` section in the example context above, the access token provided by Optimizely to your app
+lasts for 2 hours (7,200 seconds). See the FAQ below for information on what to do when this access token expires, or
+is revoked by the user.
+
+## Design Requirements
+
+To the extent possible, all apps must follow Optimizely design patterns as described at
+http://design.optimizely.com, including using the
+<a href="http://design.optimizely.com/oui/index.html" target="_blank">Optimizely User Interface (OUI)</a> custom CSS
```
Comment (jessicalc) [inline src/content/canvas/index.md]: Please change this link to https://github.com/optimizely/oui/blob/devel/README.md

Comment (jessicalc) [inline src/content/canvas/index.md]: You may consider adding a More Reading section, with links to:

- https://github.com/optimizely/oui/blob/devel/README.md (Github docs - important cos it shows them how to start using it)
- Optimizely OUI API Docs http://design.optimizely.com/oui/core/api/

**Comment by jgaul** [comment]
@jessicalc -- folks can contact us through Gitter, or using the app registration form. i'd like to rely on those channels for now.

**Pull request merged**